### DISCUSSION TITLE
X-Frame-Options deny

### DISCUSSION THREAD

**Discussion by lstrive-yn** [verdaccio/verdaccio#2102] [Q&A] [answered]
I want to use the sit built by Verdaccio via iframe for other projects in the Intranet but I got this error:

**Refused to display 'http://** . **.cc/' in a frame because it set 'X-Frame-Options' to 'deny'.**

I configured nginx to send the X-Frame-Options header, add  `SAMEORIGIN always` to server configuration but it doesn't work;

how should I do? thanks!

**Comment by DanielRuf** [comment, marked as answer]
See https://github.com/verdaccio/verdaccio/search?q=FRAMES_OPTIONS and especially https://github.com/verdaccio/verdaccio/blob/39376be34514f4c7e1d3a0f403a8dad30f7a3f72/src/api/middleware.ts#L23. We set this to prevent some types of attacks.

**Comment by lstrive-yn** [comment]
I get it, thanks a lot

发自我的iPhone
